src/modules/control_allocator/ActuatorEffectiveness/ActuatorEffectivenessTiltrotorVTOL.cpp

```diff
@@ -213,3 +220,15 @@ void ActuatorEffectivenessTiltrotorVTOL::getUnallocatedControl(int matrix_index,
 		status.unallocated_torque[2] = 0.f;
 	}
 }
+
+bool ActuatorEffectivenessTiltrotorVTOL::throttleSpoolupFinished()
+{
+	vehicle_status_s vehicle_status;
+
+	if (_vehicle_status_sub.update(&vehicle_status)) {
+		_armed = vehicle_status.arming_state == vehicle_status_s::ARMING_STATE_ARMED;
+		_armed_time = vehicle_status.armed_time;
+	}
+
+	return _armed && hrt_elapsed_time(&_armed_time) > 1_s;
```

Comment:
Can you add a comment for the 1s?

Reply:
I think we better link it to COM_SPOOLUP_TIME directly, see 78505d6675ff6af44f1794f551cc4ceee43842a8